pkg/apis/network.harvesterhci.io/v1beta1/clusternetwork.go

```diff
@@ -1,24 +1,13 @@
 package v1beta1
 
-import (
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:shortName=cn;cns,scope=Cluster
-// +kubebuilder:printcolumn:name="DESCRIPTION",type=string,JSONPath=`.spec.description`
-// +kubebuilder:printcolumn:name="ENABLE",type=boolean,JSONPath=`.enable`
 
 type ClusterNetwork struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
Still not quite sure if it's OK to remove fields in CRD.
I manually apply the updated CRD manifest and the only clusternetwork `vlan` seems to survive because most of the fields are empty.

Reply:
@yaocw2020 is the old `vlan` CR supposed to be removed in the upgrade path?

Reply:
Maybe, I am considering removing them directly or just tagging them as deprecated.

Reply:
I'll consider it in the RC2 version.